CONTRIBUTING.md

```diff
@@ -33,99 +30,119 @@ If you use [nvm](https://github.com/creationix/nvm) or [nave](https://github.com
 
 To install npm, use `npm install --global npm@^<version>`. For example, `npm install --global npm@^9.5.1`.
 
-### Android, iOS and React Native
-
-Backpack also supports React Native, plus native Android and iOS.
-
-They can be found at [backpack-android](https://github.com/skyscanner/backpack-android) and [backpack-ios](https://github.com/skyscanner/backpack-ios) and [backpack-react-native](https://github.com/skyscanner/backpack-react-native)
-
 ### Code style
 
 Backpack uses a combination of [ESLint](https://eslint.org) and [Prettier](https://prettier.io) to enforce coding styles. ESLint runs as a pre-commit hook, so it isn't possible to commit code that causes ESLint to fail.
 
 We recommend that you install [a plugin to your editor](https://eslint.org/docs/user-guide/integrations#editors) to run ESLint automatically, which will then show you any errors inline. You can even enable an option to fix ESLint errors automatically upon saving.
 
+### Android, iOS and React Native
+
+Backpack also supports React Native, plus native Android and iOS.
+
+They can be found at [backpack-android](https://github.com/skyscanner/backpack-android) and [backpack-ios](https://github.com/skyscanner/backpack-ios) and [backpack-react-native](https://github.com/skyscanner/backpack-react-native)
+
 ## Getting started
 
-**All Backpack components are written in Typescript or are being gradually migrated to Typescript.**
+Once you have a compatible environment as stated above, you can setup the project.
 
-As we're in the process of migrating all Backpack components to Typescript, we kindly request that engineers contributing changes to an existing component also migrate the respective component.
+1. Pull the code and create a new branch
 
-### Getting the code
+```sh
+git clone https://github.com/YOUR_USERNAME/Backpack.git
+git checkout -b {BRANCH_NAME}
+```
 
-You should pull code down using the following command
+2. Install npm dependencies
 
+```sh
+npm install
 ```
-git clone https://github.com/YOUR_USERNAME/Backpack.git
+
+3. Build SVGs
+
+```sh
+npm run build
 ```
 
-### Install dependencies
+4. Start the Storybook server, then go to [http://localhost:9001](http://localhost:9001) in a web browser to view it
 
-Run `npm install` to install dependencies from npm.
+```sh
+npm start
+```
 
-### Build the code
+## Write your code
 
-Backpack's code depends on some things that must be built first, such as icon fonts, SVGs and tokens.
+Before you start writing code, we recommend familiarising yourself with the engineering conventions and squad decisions which are kept in the [decisions folder](/decisions).
```

Comment:
Should we also add here to check out the guidence on the docs site too for getting started with Backpack?

https://www.skyscanner.design/latest/getting-started/about-backpack-pN209Wjo

Reply:
Hmm... feel like that page is quite generic and about the DS in general. We do hvae the principles https://www.skyscanner.design/latest/getting-started/backpack-in-code/principles-MmgbZH40 page though that could be useful to link 🤔 although they're not web only

Reply:
I think them not being web only is fine. The about backpack site is a good start, though - it also links to the engineering guide etc which is helpful